resources/models/resource.py

```diff
@@ -264,6 +266,7 @@ class Resource(ModifiableModel, AutoIdentifiedModel):
         verbose_name=_('External reservation URL'),
         help_text=_('A link to an external reservation system if this resource is managed elsewhere'),
         null=True, blank=True)
+    external_reservation_text = models.TextField(verbose_name=_('External reservation free text'), blank=True, null=True)
```

Comment:
It is not [recommended](https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.Field.null) to have a nullable `TextField` unless there is a specific need for that.

Reply:
Changed it.